contracts/src/Assets.sol

```diff
@@ -159,6 +187,42 @@ library Assets {
         emit IGateway.TokenSent(token, sender, destinationChain, destinationAddress, amount);
     }
 
+    // @dev Transfer Polkadot-native tokens back to Polkadot
+    function _sendForeignToken(
+        bytes32 foreignID,
+        address token,
+        address sender,
+        ParaID destinationChain,
+        MultiAddress calldata destinationAddress,
+        uint128 destinationChainFee,
+        uint128 amount
+    ) internal returns (Ticket memory ticket) {
+        if (destinationChainFee == 0) {
+            revert InvalidDestinationFee();
+        }
+
+        Token(token).burn(sender, amount);
+
+        ticket.dest = destinationChain;
```

Comment:
Is the dest here supposed to be `assetHubParaID`?

Reply:
Its meant to be the final destination for multi-hop transfers. 

I see now that we need to add a `SendForeignTokenToAssetHubAddress32` message type for PNA 👍

Reply:
Yeah good catch. I've reimplemented `_sendForeignToken` to mirror `_sendNativeToken`. So that PNA assets can be sent to assethub or a further hop from assethub.